exporter/trace/trace_proto.go

```diff
@@ -54,7 +54,7 @@ const (
 	labelHTTPPath       = `/http/path`
 	labelHTTPUserAgent  = `/http/user_agent`
 
-	version = "0.2.3"
+	version = "0.1.0"
```

Comment:
Is this intentional? Other places you changed from 2.3 to 3.0.

Reply:
yes. I am going to publish version 0.1.0 for stackdriver exporter.

Reply:
This version reflects stackdriver version and not the otel version.